src/timelines/main.js

```diff
@@ -9,9 +9,9 @@ import { exptBlock1, exptBlock2 } from "../config/experiment";
 
 const primaryTimeline = [
   preamble,
-  countdown({ duration: 1000, text: lang.countdown.message1, time: 3 }),
+  countdown({ duration: 1000, message: lang.countdown.message1, time: 3 }),
```

Comment:
for all of these uses, remove any keys, where our value matches the default (in the case of countdown, duration and time)